config/routes.rb

```diff
@@ -116,6 +116,10 @@
     member do
       get :schedule
     end
+
+    member do
+      get :allevents
```

Comment:
To be honest, I don't like the name allevents ... 
Do we already have a route ``/conference/<slug>/events`` ?
Any other suggestion?